src/App.js

```diff
@@ -13,25 +13,41 @@ const fetchProducts = async () => {
 
   return data;
 };
+const fetchUserData = async (token) => {
+  const data = await callApi({
+    url: "users/me",
+    token,
+  });
+  console.log("user data is:", data);
+  return data;
+};
 
 function App() {
   const [token, setToken] = useState(localStorage.getItem("token"));
   const [products, setProducts] = useState([]);
-  const [userData, setUserData] = useState([]);
+  const [userData, setUserData] = useState({});
```

Comment:
user data is stored as "user" in localStorage when you log in, can you use the getItem method to set the initial value?